apps/src/code-studio/pd/form_components/FormController.jsx

```diff
@@ -284,7 +289,7 @@ export default class FormController extends React.Component {
         style={styles.pageButtons}
         items={this.getPageComponents().length}
         activePage={this.state.currentPage + 1}
-        onSelect={i => this.setPage(i - 1)}
+        onSelect={i => this.setPage(i - 1)} // eslint-disable-line react/jsx-no-bind
```

Comment:
FYI you can get around this one too by setting a data- field to your index, and inspecting that in the callback. For example, set [here](https://github.com/code-dot-org/code-dot-org/blob/a2adb2db73b4a17d6cbc5d4cfd9145eae13ba8e1/apps/src/code-studio/pd/workshop_dashboard/workshop.jsx#L383-L390), inspected [here](https://github.com/code-dot-org/code-dot-org/blob/a2adb2db73b4a17d6cbc5d4cfd9145eae13ba8e1/apps/src/code-studio/pd/workshop_dashboard/workshop.jsx#L199-L199)